README.md

```diff
@@ -1 +1,186 @@
-# MSalyhu_portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+  <img src="./dist/img/my-profile.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Muhammad Salyhu's Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+**This is the portfolio for Muhammad Aminu Salihu as a Software Developer**
+
+## 🛠 Built With <a name="built-with"></a>
+
+1-HTML.
+2-CSS.
+3-LINTERS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://MSalyhu.com/">HTML</a></li>
+    <li><a href="https://MSalyhu.com/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Added index.html file.
+> Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> comming soon.
+
+- [Live Demo Link](https://google.com)
```

Comment:
- [ ] Kindly remove the live demo section until you deploy the app for professional documentation. 👍🏽